chalice/cli/__init__.py

```diff
@@ -182,6 +189,59 @@ def deploy(ctx, autogen_policy, profile, api_gateway_stage, stage,
     reporter.display_report(deployed_values)
 
 
+@cli.command('invoke')
+@click.option('-n', '--name', metavar='NAME', required=True,
+              help=('The name of the function to invoke. '
+                    'This is the logical name of the function. The only '
```

Comment:
Maybe rephrase the second sentence to: If the function is decorated by `app.route`, use `api_handler` as the name instead